<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>311</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="Topic1">
      <property name="text">
       <string>Topic1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Topic1Text"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>311</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="Topic2">
      <property name="text">
       <string>Topic2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Topic2Text"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>311</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="Topic4">
      <property name="text">
       <string>Topic4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Topic4Text"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>311</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QRadioButton" name="Topic3">
      <property name="text">
       <string>Topic3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Topic3Text"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Send">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Send</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>SendButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Topic1</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>Topic1Enable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Topic2</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>Topic2Enable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Topic3</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>Topic3Enable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Topic4</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>Topic4Enable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SendButtonClicked()</slot>
  <slot>Topic1Enable()</slot>
  <slot>Topic2Enable()</slot>
  <slot>Topic3Enable()</slot>
  <slot>Topic4Enable()</slot>
 </slots>
</ui>
